packages/blade-old/src/atoms/Link/Link.native.js

```diff
@@ -30,6 +31,7 @@ const Link = ({ children, onClick, size, hitSlop, testID }) => {
         <Text
           color={active ? activeTextColor : inactiveTextColor}
           size={size}
+          weight={weight}
```

Comment:
suggestion: Weight is the property of the text, not the Link. Let's say in the future we need to truncate the content of the Link, you need to add the logic for that as well in the Link component. So let remove the weight property here. @gupta-ji6

Reply:
> Weight is the property of the text, not the Link

@avinashdvv agreed, it is the property of `Text` only. The current `weight` property of `Link` passed it to `Text` only without any transformation.

What other approach would you suggest to fulfill the requirement of a `Link` with "bold" `Text`? 🤔 

Also, I couldn't really get how the truncation of the text relates to the `weight` of the `Text`. Can you share a brief example to make me understand the same? 🙈 

// cc: @chaitanyadeorukhkar